ComSemApp/templates/ComSemApp/about/base.html

```diff
@@ -43,13 +43,16 @@
 						ComSem
 					</a>
 				</p>
-				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample07" aria-controls="navbarsExample07" aria-expanded="false" aria-label="Toggle navigation">
+				<!-- Fixed hamburger buttons and changed IDs -DF -->
+				<button class="navbar-toggler" type="button" data-toggle="collapse"
+				data-target="#navbarsHamburger" aria-controls="navbarsHamburger"
+				aria-expanded="false" aria-label="Toggle navigation">
 					<span class="navbar-toggler-icon"></span>
 				</button>
 
-				<div class="collapse navbar-collapse" id="navbarsExample">
-					<ul class="navbar-nav ml-auto">
-						<li class="nav-item">
+				<div class="collapse navbar-collapse" id="navbarsHamburger">
+					<ul class="navbar-nav mr-auto">
```

Comment:
Personally I think I prefer `ml-auto` here. ml-auto makes it so the menu items float right and mr-auto floats left, eh?

ml-auto
![Screen Shot 2020-04-04 at 20 01 36](https://user-images.githubusercontent.com/17057932/78459203-44921480-76af-11ea-9005-0b07f5182656.png)

mr-auto
![Screen Shot 2020-04-04 at 20 03 55](https://user-images.githubusercontent.com/17057932/78459226-75724980-76af-11ea-9a16-6aece269d8ee.png)

I vote we leave the un-hamburgered menu the way it is.

Reply:
We agree that ml looks better, I think that was a typo from me when I was looking at a reference.